samples/initial_device_setup.yml

```diff
@@ -8,6 +8,14 @@
           objId: default
         register_as: provisionInfo
 
+    - name: Show EULA text
+      debug:
+        msg: "EULA details: {{ provisionInfo['eulaText'] }}"
+
+    - name: Confirm EULA acceptance
+      pause:
+        prompt: 'Please confirm you want to accept EULA. Press return to continue. To about, press Ctrl+C and then "A"'
```

Comment:
To abort?

Reply:
we have all key names in this sentence define with a first capital letter - should we apply the same for return key name?

Reply:
Good catch, fixed.